### DISCUSSION TITLE
[NEXT-784] Strange session Dropdown in analytics sessions after upgrade to 13.2

### DISCUSSION THREAD

**Discussion by gruelt** [vercel/next.js#47901] [Help]
### Verify canary release

- [ ] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
Operating System:
      Platform: linux
      Arch: x64
      Version: #33~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Mon Jan 30 17:03:34 UTC 2
    Binaries:
      Node: 16.19.0
      npm: 8.19.3
      Yarn: N/A
      pnpm: N/A
    Relevant packages:
      next: 13.0.1
      eslint-config-next: 13.0.0
      react: 18.2.0
      react-dom: 18.2.0
```

### Which area(s) of Next.js are affected? (leave empty if unsure)

Metadata (metadata, generateMetadata, next/head, head.js)

### Link to the code that reproduces this issue

analytics.com

### To Reproduce

Upgrade from 13.1 to 13.2

Go to google analytics : 

In google campaigns , entering clicks are still at the same level , but "sessions" has a dropdown of -60%.
Clics : 
![clics](https://user-images.githubusercontent.com/5389596/222764255-887e8d3c-6324-4d26-ade2-cb0b099e1e7b.png)
Sessions ( with a clear dropdown )
![sessions](https://user-images.githubusercontent.com/5389596/222764408-e06a1eef-c79a-4ebd-aa6d-93a460bb5838.png)

Observed on two different websites ( images representing only one)  , with each time a drop down the day it was released with version 13.2

### Describe the Bug

Analytics metric "sessions" os droped down by -60 % since released , but other metrics are not  moving.
GTM is used

### Expected Behavior

Having a coherent data in analytics.

### Which browser are you using? (if relevant)

Version 109.0.5414.119 (Build officiel) (64 bits)

### How are you deploying your application? (if relevant)

aws beanstalk

<sub>[NEXT-784](https://linear.app/vercel/issue/NEXT-784/strange-session-dropdown-in-analytics-sessions-after-upgrade-to-132)</sub>

**Comment by timneutkens** [comment]
Converted to Discussions given that the reproduction steps don't include a url we can look at and assumes we can log into your google analytics property, which we can't, so there's no way to investigate this further.

My assumption given that tagmanager is mentioned is that your site is somehow breaking React hydration by injecting additional scripts, but that can't be verified because there is no reproduction included.
